instrumentation/jaxrs-client/jaxrs-client-1.1/src/main/java/io/opentelemetry/instrumentation/auto/jaxrsclient/v1_1/InjectAdapter.java

```diff
@@ -16,15 +16,15 @@
 
 package io.opentelemetry.instrumentation.auto.jaxrsclient.v1_1;
 
-import com.sun.jersey.api.client.ClientRequest;
 import io.opentelemetry.context.propagation.HttpTextFormat;
+import javax.ws.rs.core.MultivaluedMap;
 
-public final class InjectAdapter implements HttpTextFormat.Setter<ClientRequest> {
+public final class InjectAdapter implements HttpTextFormat.Setter<MultivaluedMap<String, Object>> {
 
   public static final InjectAdapter SETTER = new InjectAdapter();
 
   @Override
-  public void set(ClientRequest clientRequest, String key, String value) {
-    clientRequest.getHeaders().putSingle(key, value);
+  public void set(MultivaluedMap<String, Object> carrier, String key, String value) {
```

Comment:
I am not sure this change made things better. Why replaces `ClientRequest` with `MultivaluedMap<String, Object>` here?

Reply:
same

Reply:
and done